src/main/resources/view/HelpWindow.fxml

```diff
@@ -1,60 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import com.jfoenix.controls.JFXButton?>
 <?import java.net.URL?>
 <?import javafx.geometry.Insets?>
-<?import javafx.scene.control.Button?>
+<?import javafx.scene.Scene?>
 <?import javafx.scene.control.Hyperlink?>
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.HBox?>
 <?import javafx.scene.layout.VBox?>
-<?import javafx.scene.Scene?>
 <?import javafx.stage.Stage?>
 
-<fx:root resizable="false" title="Help [Press Esc to close]" type="javafx.stage.Stage"
-         xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root resizable="false" title="Help [Press Esc to close]" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1">
     <icons>
-        <Image url="@/images/help_icon.png"/>
+        <Image url="@/images/help_icon.png" />
```

Comment:
For all the changes in this file, should probably standardize to no space before `/>`, since thats the format of all the fxml files

Reply:
Also, this help icon should be added to the top context menu (can do in a later issue)